.github/workflows/docs.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches:
```

Comment:
We could also rename the job from `Docs` to `Docs v1` or something similar.

Reply:
Good point, see https://github.com/keep-network/keep-core/pull/2997.